### DISCUSSION TITLE
Does running Fruchterman Reingold layout takes too much time?

### DISCUSSION THREAD

**Discussion by sandipansamaddar** [gephi/gephi#2323] [Q&A]
Hello,
I am trying to visualize a microbial interaction network which has 73 nodes and 200 edges. After exporting the files when I am selecting the Fruchterman Reingold layout the task keeps on running. I ran it overnight in both my windows and mac but it never finished. What can be the issue? Or does it need it stopped manually when I am satisfied with the pattern? Partial node list and edge list files are attached if anyone wants to check whether the format is correct or not. Any help will be really appreciated.

Thanks,

Sandipan

[Nodelist.txt](https://github.com/gephi/gephi/files/6541269/Nodelist.txt)
[edgelist.txt](https://github.com/gephi/gephi/files/6541270/edgelist.txt)

**Comment by martinastoycheva** [comment]
Hell Sandipan, 

I ahve the same issue. Did you find an answer? 

Cheers, 
Martina

**Reply by mbastian** [reply]
Hi, the algorithm is not meant to stop by itself. Stop it yourself when you're satisfied with the layout. This can vary a lot between graphs so that's why automated stopping is actually a challenging problem to solve.

**Reply by martinastoycheva** [reply]
Thank you for the rapid response!
